mteb/tasks/Retrieval/rus/MMarcoRetrieval.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class RuMMarcoRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="RuMMarcoRetrieval",
+        dataset={
+            "path": "ai-forever/ru-mmarco-retrieval",
+            "revision": "18d1c2b1ab2a7e8920614329e19ab4c513113d7e",
+        },
+        description="mMARCO: A Multilingual Version of the MS MARCO Passage Ranking Dataset",
+        reference="https://arxiv.org/abs/2108.13897",
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["dev"],
+        eval_langs=["rus-Cyrl"],
+        main_score="ndcg_at_10",
+        date=("2000-01-01", "2019-01-01"),
+        form=["written"],
+        domains=["Web"],
+        task_subtypes=["Question answering"],
+        license="apache-2.0",
+        socioeconomic_status="mixed",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="machine-translated",
```

Comment:
This datasets obtained from msmarco which is widely used for evaluation, but is only available in english. Other languages usually have no option to evaluate retrieval capabilities of models. We have miracl and rubq only, both wiki-based and that is all. MMarco was added to C-MTEB and PL-MTEB which we used as reference. Also, the dataset was translated using Google API, which one is pretty good for the Russian language. What's your thoughts?

Reply:
Hmm given the remainder of the datasets, I don't think it is needed. I believe that MSMarco either correlates enough with your existing tasks to be redundant or if it does not I would be afraid that it is due to translation artifacts.

that being said I am up for accepting it if you feel strongly about it (to construct a Russian benchmark) but it will not be included in the MMTEB points and probably not in multilingual benchmark datasets.